scripts/binary/bump.js

```diff
@@ -19,13 +19,6 @@ const _PROVIDERS = {
       'cypress-io/cypress-test-module-api',
       'cypress-io/cypress-test-node-versions',
```

Comment:
long terem, will the plan be to remove all of these projects from bump.js to the circleci jobs or maybe expand to these repos to run for linux, darwin and win32?

Reply:
I'd like to eventually get rid of this "bump" stuff altogether. We can test almost everything inside of this monorepo's CircleCI where it's much easier for developers to debug failures/add new tests/understand the system. The only thing that `bump` adds is the ability for us to e2e test Cypress running in response to a git push that triggered a CI run. Theoretically this would allow us to test against multiple CI providers, but practically, we don't run against multiple CI providers in CI anyways because it was too much to maintain. Plus, we have plenty of tests that cover specific CI provider functionality in the `server`.

I'm spiking into extending system-tests to run against built Cypress and/or with Docker containers, so that we don't need `bump.js` to run node-versions and ci-environments.

Reply:
it'd be amazing if we could drop this! It's def a hidden test case, especially since it only runs on a few branches.